docs/lib/auth/fragments/native_common/signin/common.md

```diff
@@ -117,3 +117,9 @@ As a result, for testing purposes, you'll at least need an input field where you
 <inline-fragment platform="ios" src="~/lib/auth/fragments/ios/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
 <inline-fragment platform="android" src="~/lib/auth/fragments/android/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
 <inline-fragment platform="flutter" src="~/lib/auth/fragments/flutter/signin/50_multi_factor_confirm_signin.md"></inline-fragment>
+
+## Check user sign-in state
```

Comment:
Since this applies to all types of sign in (Sign In, Sign In with web UI, Social sign in with web UI), can we create a new page for this instead of adding it to the existing "Sign In" page?    I'm thinking it could be a page named "Check sign in state", right after the "Sign in next steps" page.

Reply:
Yeah, this is reasonable 👍